### DISCUSSION TITLE
How do I disable "Vercel Protection" for by default for my team?

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/community#5420] [Help] [answered]
### Summary

All our deployments used to be public, but recently something changed and now we get this for Preview deployment:
![image](https://github.com/vercel/community/assets/3807458/661cd8c6-2d0c-48b0-8c99-b2caae4f7d36)

I believe it's related to a new feature "Deployment Protection" and more specifically "Vercel Authentication", but we don't want it for a specific team.
https://vercel.com/docs/security/deployment-protection#understanding-deployment-protection-by-environment

How do I disable it by default?

I managed to disable it per-project under the project settings.
![image](https://github.com/vercel/community/assets/3807458/838f5da8-41c3-4f30-8d3b-35894e5e5375)

But I need to disable that by default, because we need public access for all our deployments for that team.
If it can be done through a Team setting (ideally, but haven't found such setting), or ENV variable, that would be great.

Related documentation:
- https://vercel.com/docs/security/deployment-protection/methods-to-bypass-deployment-protection/protection-bypass-automation
- https://vercel.com/docs/security/deployment-protection/methods-to-protect-deployments/vercel-authentication

### Steps to Reproduce

None.

**Comment by arzafran** [comment]
Ended up here thinking it's the only way to make GitHub Actions work with Vercel?
we run lighthouse on every deployment but Vercel Authentication on by default broke our script,
manually disabling per project works, but would also need to do in every project, is there a way to set it as default OR bypass it entirely?

**Comment by amyegan** [comment, marked as answer]
Thanks for letting us know this is an option you need! We don't currently have a way to disable Vercel Authentication for a team by default, but our engineers are working on plans to provide that option. 

For now you would need to disable it in project settings as you've already done.

**Reply by Vadorequest** [reply]
Thanks for taking this into consideration, I'm pretty sure more and more Vercel customers will need to opt-out from this feature as a default.

**Comment by abhi-micra** [comment]
![Screenshot 2024-08-05 at 10 35 01 AM](https://github.com/user-attachments/assets/a8ab9c0e-efef-4cd5-9a85-fc0348af3b90)

### Project Settings

1. Deployment Protection
2. Vercel Authentication -> Disabled ( set )

**Reply by jaiminmicra** [reply]
Yes, It's working,
Thanks dude!

**Reply by pencilcheck** [reply]
Thanks!

[a comment by an automated account is not shown]
